znoyder/config.yml

```diff
@@ -167,6 +167,12 @@ add:
         pipeline:
           - 'gate'
 
+  'openstack-tempest':
+    'osp-17.0':
+      'osp-tox-py39':
+        pipeline:
+          - 'gate'
```

Comment:
I think you are more interested in having 'check' here.
The gating is in the roadmap, but not yet implemented in out downstream Gerrit.